projects/core/src/cart/facade/cart-data.service.ts

```diff
@@ -1,32 +1,41 @@
 import { Injectable } from '@angular/core';
```

Comment:
Now, we need add unit test for it.

Reply:
If not a problem, we postponed adding unit tests to separate ticket https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/3242